Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1481 @@
+    Returns:
+        A CommandResult object with a list of the comments or a single comment on a specific issue.
+    """
+    repo = args.get('repo')
+    pr_id = int(args.get('pull_request_id', ''))
+    comment_id = args.get('comment_id', None)
+    limit = int(args.get('limit', 50))
+    page = int(args.get('page', 1))
+    check_args(limit, page)
+    page_size = min(100, limit)
+    params = {
+        'page': page,
+        'pagelen': page_size
+    }
+    if not repo:
+        repo = client.repository
+    response = client.pull_request_comment_list_request(repo, pr_id, params, comment_id)
+    if comment_id:
+        results = [response]
+        hr_title = f'The information about the comment "{comment_id}"'
+    else:
+        results = check_pagination(client, response, limit)
+        hr_title = f'List of the comments on pull request number "{pr_id}"'
+    human_readable = []
+    records_to_delete = []
+    for value in results:
+        if not value.get('content', {}).get('raw') == "":
+            d = {'Id': value.get('id'),
+                 'Content': value.get('content', {}).get('raw'),
+                 'CreatedBy': value.get('user', {}).get('display_name'),
+                 'CreatedAt': value.get('created_on'),
+                 'UpdatedAt': value.get('updated_on'),
+                 'PullRequestIdIssueId': value.get('pullrequest', {}).get('id'),
+                 'PullRequestTitle': value.get('pullrequest', {}).get('title')}
+            human_readable.append(d)
+        else:
+            records_to_delete.append(value)
+    if len(records_to_delete) > 0:
+        for item in records_to_delete:
+            results.remove(item)
+
+    headers = ['Id', 'Content', 'CreatedBy', 'CreatedAt', 'UpdatedAt', 'IssueId', 'IssueTitle']
+    readable_output = tableToMarkdown(
+        name=hr_title,
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.PullRequestComment',
+        outputs=results,
+        raw_response=results
+    )
+
+
+def pull_request_comment_update_command(client: Client, args: Dict) -> CommandResults:
+    """ Updates a comment in a pull request.
+        Args:
+            client: A Bitbucket client.
+            args: Demisto args.
+        Returns:
+            A CommandResult object with a success message.
+    """
+    repo = args.get('repo')
+    pr_id = int(args.get('pull_request_id', ''))
+    comment_id = args.get('comment_id', '')
+    content = args.get('content')
+    if not repo:
+        repo = client.repository
+    body = {
+        'content': {
+            'raw': content
+        }
+    }
+    response = client.pull_request_comment_update_request(repo, pr_id, body, comment_id)
+    return CommandResults(
+        readable_output='The comment was updated successfully',
+        outputs_prefix='Bitbucket.PullRequestComment',
+        outputs=[response],
```

Comment:
Why the list here in the outputs and raw_response?

Reply:
I don't remember. I removed it and it works.